README.md

```diff
@@ -93,195 +93,38 @@ Mapbox has a free tier, however there are paid options available. For most users
-     ​
-2. Install node.
-   ​
-   - macOS: `brew install node@14`
-   - Ubuntu: `sudo apt install nodejs`
-     ​
-3. `yarn install` will install everything in `package.json`.
-   ​
+The provided `docker-compose.yml` file in the root directory of the project is intended to assist developer in quickly getting a locally running instance of Harmony up along with several services it relies on (postgres, redis & hasura). The local filesystem is mounted, so any changes made to the codebase will be reflected in the running web container.
 
 ### Druid setup
-
 ​
 Specify druid host in `global_config.py`: `DEFAULT_DRUID_HOST = '<public production Druid server IP>'`
+### Prepare environment
 
+1. Install the latest version of [Docker](https://docs.docker.com/get-docker/). 
+2. Clone the git repository: `git clone https://github.com/Zenysis/Harmony`. (Alternatively, you may want to fork the repo and clone the fork — that way you can use version control for your customization.)
+3. Build your development docker images: `docker compose build` (this will take some time!)
 
-### PostgreSQL
-
-You will have to set up a [PostgreSQL database](https://www.postgresql.org/) to host relational data and web application state.
-
-1.  Install postgres.
-
-    - macOS: `brew install postgresql`
-    - Ubuntu: `sudo apt install postgresql`
-
-2.  Start postgres server: `./scripts/db/postgres/dev/start_postgres.sh`
-
-3. [Ubuntu only as this is the default setup on macOS]: The postgresql authentication rules will need to change to allow easier access and the server to connect to postgres. This requires editing the `pg_hba.conf` file and restarting postgres. Here is more information about the configuration file and what changes are being made https://www.postgresql.org/docs/current/auth-pg-hba-conf.html.
-
-   Find the configuration file by running `sudo -u postgres psql -c "SHOW hba_file;"`. Edit the first uncommented line of that file to be
-      ```
-      local   all             postgres                                trust
-      ```
-   Then, add the following lines at the bottom:
-      ```
-      # Allow any user on the local system using local loopback TCP/IP connections
-      # to connect to any database with any database user name.
-      host    all             all             127.0.0.1/32            trust
-      # The same over IPv6.
-      host    all             all             ::1/128                 trust
-      ```
-   Finally, run `sudo service postgresql restart` for the changes to take effect.    ​
-
-4.  Enter psql client to check server success. If it works, then you can exit psql.
-
-    - macOS: `psql postgres`
-    - Ubuntu: `psql -U postgres`
-
-5.  Create and upgrade the local postgres database as well as populate the Data Catalog tables: `./scripts/db/postgres/dev/init_db.py --populate_indicators`
```

Comment:
Adding this step back in could fix the ordering issue. It also handles getting the dimensions populated in Data Catalog, but it's not something that should run every time because it will overwrite all of Data Catalog and read it in from the code.

Reply:
Added it in as manual step in README